packages/cf-util-http/src/http.js

```diff
@@ -184,10 +184,7 @@ export function request(method, url, opts, callback) {
 
         return response.text().then(text => {
           if (callback) {
-            callback(
-              undefined,
-              wrapResponse(headers, status, text, response)
-            );
+            callback(undefined, wrapResponse(headers, status, text, response));
```

Comment:
I took the `|| true` from the lint script and am running prettier on every file I change so it formatted this line correctly and am no longer allowing builds where prettier/eslint errors occur.